modules/ascat/main.nf

```diff
@@ -85,16 +87,16 @@ process ASCAT {
 
     #Load the data
     ascat.bc = ascat.loadData(
-        Tumor_LogR_file = "Tumour_tumourLogR.txt",
-        Tumor_BAF_file = "Tumour_normalBAF.txt",
-        Germline_LogR_file = "Tumour_normalLogR.txt",
-        Germline_BAF_file = "Tumour_normalBAF.txt",
+        Tumor_LogR_file = paste0("$prefix", "_tumour_tumourLogR.txt"),
+        Tumor_BAF_file = paste0("$prefix", "_tumour_tumourBAF.txt"),
```

Comment:
why do we need tumour twice?

Reply:
yes, at least if we want to output all possible output files